### PR TITLE
infer_android_step 1.0.0

### PR DESCRIPTION
**Pull request by ataourirt**
This is a Android step for [Infer](http://fbinfer.com)

### CODE REVIEW

**Comment by godrei** [comment]
Thank you very much!
We will review it ASAP. Probably on tomorrow.

File: steps/infer_android_step/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+title: Infer Android Step
+summary: Step for Bitrise
+description: |-
+  This is a Step for [FbInfer App](http://fbinfer.com).
+  This is not an officially supported plugin
+website: https://github.com/ataourirt/fbinfer-android-bitrise
+source_code_url: https://github.com/ataourirt/fbinfer-android-bitrise
+support_url: https://github.com/ataourirt/fbinfer-android-bitrise
```
Comment (godrei) [inline steps/infer_android_step/1.0.0/step.yml]: Can you please change this to `https://github.com/ataourirt/fbinfer-android-bitrise/issues`?

**Comment by godrei** [comment]
Hi @ataourirt !
Currently android sdk and tools are only available on the `linux` and `xamarin` stack, but you set `project_type_tags: ios`, is `infer` working whithout this environment?

**Comment by mackoj** [comment]
Thanks @godrei,

@ataourirt does not work today he will do more test soon and try it on other environment and fix your requests.

**Comment by godrei** [comment]
Hi @mackoj and @ataourirt ,
i'm just curious if you want to finish this step share, or should we close this PR?

**Comment by mackoj** [comment]
Infer is in pause actually because of issue on the tool. So we will close it.

**Pull request closed without merging**